scripts/inference.py

```diff
@@ -26,6 +26,30 @@
     required=True,
     help="Path to the directory containing LLaMa weights (.pth files sharded by tensor parallel rank, not HF weights)",
 )
+parser.add_argument(
```

Comment:
I'd check, as I think current might have some equal or really similar parameters, to make sure we aren't doubling parameters with similar names